CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates how the `DatePicker` and `Slider` components internally use `TextInput`.
 - Updates how the `Pagination` component internally uses `Link`.
 - Updates how the `Tabs` component internally uses `Button`.
+- Updates how images are passed to the `Hero` component. Instead of `imageAlt` and `imageSrc` props, now an object must be passed to the `imageProps` prop.
```

Comment:
You did not include anything about the updated docs. Was that intentional?

Reply:
Yeah. We don't typically include documentation updates in the changelog.